internals/api/dirfs_test.go

```diff
@@ -23,21 +23,21 @@ func TestAbsDirPath(t *testing.T) {
 	dir := &Dir{
 		DirID:    uuid.New(),
 		Name:     "dir",
-		ParentID: repoDir.DirID,
+		ParentID: &repoDir.DirID,
 	}
 
 	subdir := &Dir{
 		DirID:    uuid.New(),
 		Name:     "subdir",
-		ParentID: dir.DirID,
+		ParentID: &dir.DirID,
```

Comment:
Why is this still a pointer?

Reply:
Because the parent ID is optional. It is not set for root directories.